src/Test/MakerTestEnvironment.php

```diff
@@ -403,6 +403,12 @@ private function determineMissingDependencies(): array
 echo json_encode($missingDependencies);
         ');
 
+        // The "git clone" command do not copy vendor/ dir because it's in the .gitignore.
+        // Copy vendor/ dir manually if it does not exist before running "dep_runner.php"
+        // that requires vendor/autoload.php
+        if (!$this->fs->exists($this->path.'/vendor')) {
```

Comment:
Or do you think we need one more extra check here to make sure it's only running on Mac OS if on other platforms it somehow works?

Reply:
IIRC this git stuff was added to speed up local test execution, CI always worked faster